src/cli.js

```diff
@@ -72,9 +72,7 @@ if (verbose) {
 concurently(commands, options)
     .then(() => {
         logSuccess('\ndone');
-    })
-    .catch(error => {
-        logError('\nTasks exited with error :');
-
-        throw error;
+    }, (failure) => {
+        logError('\nOne of tasks returned error');
+        process.exit(failure);
```

Comment:
die `failure`. Is dat een exit code? Of een string message?

Reply:
exit code

Reply:
ok, misschien wel goeie om even te hernoemen dan. Was voor mij niet duidelijk iig.